src/data/rustfmt.yaml

```diff
@@ -0,0 +1,4 @@
+name: rustfmt
```

Comment:
can we use pascal case here (first letter capitalized) to make it look similar to other project?